<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistics</class>
 <widget class="QWidget" name="Statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #e6f0fa; /* Fond bleu trÃ¨s clair */
    font-family: 'Segoe UI', sans-serif;
    font-size: 14px;
}
</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Main title */
QLabel#titleLabel {
    font-size: 24px;
    font-weight: bold;
    color: #1F3C88;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, 
                              stop:0 rgba(255,255,255,0.9), 
                              stop:1 rgba(245,249,255,0.8));
    border-bottom: 2px solid #2A5D9F;
    padding: 12px 20px;
    margin-bottom: 15px;
}</string>
   </property>
   <property name="text">
    <string>                  ðŸ“Š Statistics</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>971</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    font-size: 18px;
    font-weight: bold;
    color: #607D8B;
    border: 1px solid #B0BEC5;
    border-radius: 4px;
    margin-top: 20px;
    padding-top: 25px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    left: 10px;
}</string>
      </property>
      <property name="title">
       <string>ðŸ“ˆ Summary</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="successRateLabel">
         <property name="styleSheet">
          <string notr="true">QLabel#successRateLabel {
    font-size: 18px;
    font-weight: bold;
    color: #113f7c; /* Bleu profond */
}
</string>
         </property>
         <property name="text">
          <string>Success Rate :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="averageDurationLabel">
         <property name="styleSheet">
          <string notr="true">QLabel#averageDurationLabel {
    font-size: 18px;
    font-weight: bold;
    color: #113f7c; /* Bleu profond */
}
</string>
         </property>
         <property name="text">
          <string>Average Duration :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    font-size: 18px;
    font-weight: bold;
    color: #607D8B;
    border: 1px solid #B0BEC5;
    border-radius: 4px;
    margin-top: 20px;
    padding-top: 25px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    left: 10px;
}</string>
         </property>
         <property name="title">
          <string>âš–Comparison</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="comparisonTypeComboBox">
            <property name="styleSheet">
             <string notr="true">QComboBox {
    border: 1px solid #1F3C88;
    border-radius: 4px;
    padding: 6px;
    background: #F5F9FF;
}</string>
            </property>
            <item>
             <property name="text">
              <string>By Responsible</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>By Protocol</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="comparisonTable">
            <property name="styleSheet">
             <string notr="true">/* Style pour la table entiÃ¨re */
QTableWidget {
    border: 1px solid #113f7c;
    border-radius: 3px;
    gridline-color: #c0c0c0;
    font-family: &quot;Segoe UI&quot;;
    font-size: 13px;
}

/* Style pour l'en-tÃªte des colonnes */
QHeaderView::section {
    background-color: #113f7c;
    color: white;
    padding: 8px 12px;
    border: none;
    font-weight: bold;
    font-size: 12px;
}

/* Style pour les cellules de la table */
QTableWidget QTableCornerButton::section {
    background: #113f7c;
    border: none;
}

QTableView::item {
    padding: 6px;
    border-bottom: 1px solid #e0e0e0;
}

/* Style pour les cellules sÃ©lectionnÃ©es */
QTableView::item:selected {
    background-color: #113f7c;
    color: white;
}

/* Style au survol */
QTableView::item:hover {
    background-color: #d0e0f0;
}

/* Style alternÃ© pour les lignes */
QTableView::item:nth-child(even) {
    background-color: #f8f8f8;
}

QTableView::item:nth-child(odd) {
    background-color: white;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshStatsButton">
            <property name="styleSheet">
             <string notr="true">QPushButton#refreshStatsButton {
	background-color: #113f7c; 
    padding: 8px 16px;
    border-radius: 6px;
	font-size: 18px;
    font-weight: bold;
    color: white;
}

QPushButton#refreshStatsButton:hover {
    background-color: #0e3262;
}
</string>
            </property>
            <property name="text">
             <string>ðŸ”„ Refresh Statistics</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
